Refit/RequestBuilderImplementation.cs

```diff
@@ -170,8 +170,16 @@ void AddMultipartItem(MultipartFormDataContent multiPartContent, string fileName
                             return (T)(object)await reader.ReadToEndAsync().ConfigureAwait(false);
                         }
 
-                        using(var jsonReader = new JsonTextReader(reader))
+                        using (var jsonReader = new JsonTextReader(reader))
                         {
+                            if (restMethod.SerializedReturnType.IsConstructedGenericType &&
+                                restMethod.SerializedReturnType.GetGenericTypeDefinition() == typeof(RefitResponse<>).GetGenericTypeDefinition())
```

Comment:
The `GetGenericTypeDefinition()` call on `typeof(RefitResponse<>).GetGenericTypeDefinition()` is a noop -- `typeof(RefitResponse<>)` is already a generic type definition.

Reply:
Nice! I made the changes as suggested.